### ISSUE TITLE
Pre-training vs fine-tuning?

### ISSUE THREAD

**Issue by kalyanks0611** [dmis-lab/biobert#46]
First thanks for publicly releasing BioBERT model. I have  read your paper  titled, "Pre-trained Language Model for Biomedical Question Answering"  (https://arxiv.org/abs/1909.08229) and it is nice.

In the paper,  it was mentioned like 

1)  We leverage BioBERT to address these issues. To mitigate the small size  of datasets, we first **fine-tune BioBERT on other large-scale extractive question answering datasets, and then fine-tune it on BioASQ datasets**

2) **BioBERT first pre-trained on SQuAD and then fine-tuned on BioASQ** 6b obtained the best
performance over other two experiments, demonstrating the effectiveness of pretraining BioBERT on SQuAD, a comprehensive and large-scale question answering corpus.

In 1) you have mentioned like, "**fine tuning on SQuAD and then fine tuning on BioASQ**", in 2) you have mentioned  like, "**first pre-trained on SQuAD and then fine-tuned on BioASQ**".

In general,  **pre-training means, BERT model is trained on unlabeled text using Masked Language Model and Next Sentence Prediction tasks**. **Fine-tuning means, a task specific layer is added on the top of BERT and all the parameters are fine-tuned using labeled data set**

I would like to know,  **whether you have first pre-trained or fine-tuned BioBERT on SQuAD?**

Thanks in advance. Correct me if i'm wrong.

**Comment by jhyuklee** [comment]
Hi, thank you for your interest in our work.

Generally, the term "pretraining" depends on the context. When you finetune something, then everything else trained before would become pretrained components.

To answer your question based on your definition, it is finetuned on Squad before Bioasq.
After pretraining BioBERT on pubmed corpora, it is finetuned in two stages.
First on squad, and then bioasq.
In the perspective of the second stage (bioasq fine tuning), squad training would be pretraining (not related to the MLM or next sent pred. in BERT).

The writing might be confusing, but you can just see this as two stage finetuning.
Thanks.

**Comment by kalyanks0611** [comment]
Thanks for the clarification.

**Comment by kalyanks0611** [comment]
One more clarification, I need.
**For fine tuning on Squad, whether you have used only training set instances or training +test set instances**?

**Comment by wonjininfo** [comment]
Thanks Jinhyuk!
Hi @kalyanks0611, we only used training set.
